apps/rewards/app/utils/helpers.js

```diff
@@ -1,6 +1,12 @@
 import { ETH_DECIMALS } from './constants'
+import BN from 'bignumber.js'
```

Comment:
I think it's probably best to actually import this as BigNumber since BN is sort of a different library. This might be overly semantic?